### DISCUSSION TITLE
How to add another map?

### DISCUSSION THREAD

**Discussion by zuramai** [themustafaomar/jsvectormap#102] [Q&A]
I have Indonesia's map that I got from https://commons.wikimedia.org/wiki/File:Indonesia_location_map.svg

How can I add the map into jsvectormap? Should I get every province's `<path d="">` values and map it into object?

**Comment by themustafaomar** [comment]
Well, it will work! but will lose some important features such as markers, lines and more.

In order to generate a working map you have to use jvectormap converter, unfortunately the converter was abandoned a long time ago, you may encounter some dependency-issues related to Python.
 
Please take a look at: https://jvm-docs.vercel.app/docs/available-maps#generating-maps

Related issues: #80 #61
